tests/test_opening_hours.py

```diff
@@ -0,0 +1,47 @@
+from app import app
+from freezegun import freeze_time
+from apistar.test import TestClient
+from idunn.blocks.opening_hour import OpeningHourBlock
+
+def test_opening_hour():
+    """
+    Test that the opening_hours block for a POI located
+    in a different timezone (Moscow) contains the correct
+    information.
+    """
+    client = TestClient(app)
+    with freeze_time("2018-06-14 8:30:00", tz_offset=0):
+        """
+        We freeze time at 8h30 UTC (ie. 11h30 Moscow)
+        The POI located in Moscow should be open since
+        it opens at 10h00 and the local time is 11h30.
+
+        Below we create a fake OpeningHourBlock from a
+        raw json extracted from a POI located in
+        Moscow city that opens at 10h00.
+        """
+        oh_block = OpeningHourBlock.from_es(
+            {
+                "coord": {
+                    "lon": 37.588161523500276,
+                    "lat": 55.74831406552745
+                },
+                "properties": {
+                    "opening_hours": "Mo-Su 10:00-22:00"
+                }
+            },
+            lang='en'
```

Comment:
do we need the lang ?

Reply:
Yes since 5971d611d6c1a1df7f9eaabd9ab6eb8110496f45 it's mandatory.